config/core/manager_role_aggregation_patch.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      cluster.x-k8s.io/aggregate-to-manager: "true"
```

Comment:
Do we want the empty string here instead of true?

Reply:
If we do that, I think we'd have to switch `ClusterRoleSelectors` to be:
```yaml
- matchExpressions:
  - key: cluster.x-k8s.io/aggregate-to-manager
    operator: Exists
```
Instead of using `matchLabels`

I do like the idea of just checking for the existence of the label instead of requirement a specific value, though.

Reply:
I went with `true` just to be consistent with the [example in the docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) but I do not mind